### PR TITLE
[#167102226] Added component for list services organizations

### PR DESCRIPTION
**Pull request by laurapugliese**
Added component for list services organizations

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
## Affected stories

  * 🌟 [#167102226](https://www.pivotaltracker.com/story/show/167102226): Come CIT voglio visualizzare un componente che fornisca la lista degli enti che possono essere visualizzati nella sezione locali

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by laurapugliese** [comment]
componente

Jul-09-2019 14-53-36

File: ts/components/ChooserListComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
+import I18n from "i18n-js";
+import { Container, Content, View } from "native-base";
+import * as React from "react";
+import { FlatList, ListRenderItem, StyleSheet } from "react-native";
+import customVariables from "../theme/variables";
+import AppHeader from "./ui/AppHeader";
+import FooterWithButtons from "./ui/FooterWithButtons";
+
+type Props = {
+  items: ReadonlyArray<any>;
```
Comment (fpersico) [inline ts/components/ChooserListComponent.tsx]: Do not use "any". Try to create a [Generic type](https://www.typescriptlang.org/docs/handbook/generics.html).

Comment (laurapugliese) [reply ts/components/ChooserListComponent.tsx]: @fpersico Done in commit 835dd12

**Pull request merged**